### PR TITLE
chore(configs): support dependabot conventional commits

### PR DESCRIPTION
**Pull request by TheSpiritXIII**

### CODE REVIEW

File: .conform.yaml [hunk 1]
```diff
@@ -7,23 +7,26 @@ policies:
         case: lower
         invalidLastCharacters: .
       body:
-        required: true
+        required: false
```
Comment (bernot-dev) [inline .conform.yaml:10]: Question: Dependabot doesn't create commit bodies?

Comment (TheSpiritXIII) [reply .conform.yaml:10]: Dependabot includes commit bodies. This is more of a me change. I don't create commit bodies so this exact commit would have failed without this change. Thoughts?

Comment (bernot-dev) [reply .conform.yaml:10]: It looks to me like Dependabot creates well-formed commit bodies with a DCO and everything. Is this unrelated?

Comment (TheSpiritXIII) [reply .conform.yaml:10]: Correct. I can split this out if requested.

Comment (bernot-dev) [reply .conform.yaml:10]: Yeah, there's probably not a good reason for us to enforce commit bodies at the moment.

**Review by bernot-dev — APPROVED** [review]
LGTM.

**Pull request merged**